openedx/features/discounts/applicability.py

```diff
@@ -82,6 +82,30 @@ def get_discount_expiration_date(user, course):
     return discount_expiration_date
 
 
+def AA759_can_show_coupon(user, course):
+    """
+    Check whether this combination of user and course
+    can receive the AA-759 experiment discount.
+    """
+    # Course end date needs to be in the future
+    if course.has_ended():
+        return False
+
+    # Course needs to have a non-expired verified mode
+    modes_dict = CourseMode.modes_for_course_dict(course=course, include_expired=False)
+    verified_mode = modes_dict.get('verified', None)
+    if not verified_mode:
```

Comment:
tiny nit: Might be slightly more short/readable to do `if 'verified' not in modes_dict:`

Do we also want to confirm that there's an audit track though? Verified-only courses are a thing. And we should be confirming that the user is in the audit track here, right?

Reply:
sure
yeah good point, will add a check for audit